### DISCUSSION TITLE
Sofar Solar HYD10 + CHINT DTSU666 I cannot connect with inverter

### DISCUSSION THREAD

**Discussion by TwardyBobek** [wills106/homeassistant-solax-modbus#541] [Q&A]
I cannot connect with inverter I have lost one day with this problem I don't know what can I do.
With Node Red I can connect with smart metrer CHINT DTSU666. And with inverter is not possible to connect. maybe have it someone idea what can I do.

![2023-09-17 193227](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/1867e4e0-8f6a-41be-ba53-389f2408faa4)
![2023-09-17 193419](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/34eca810-2615-4912-83e9-0350c53ef59d)
![2023-09-17 193743](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/f6bd9d9c-f205-455b-9aab-eb74353749b3)
![2023-09-17 193106](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/c9aa3595-0368-4455-bc16-80d2341883a6)
![20230917_195153](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/3fcfb28d-0e9e-4c9f-9b14-f80547c69c83)
![20230917_194103](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/6900d872-b780-492b-8767-df9ed76f96ea)
![20230917_200510](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/45c049e1-3e95-47be-9c1e-103d51c8e7d5)
![20230917_195347](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/94ba2b14-6070-48eb-9cc0-76589abbc58e)
![20230917_195358](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/486d406c-2de8-4922-bf4d-7555591d94d2)
![2023-09-17 191017](https://github.com/wills106/homeassistant-solax-modbus/assets/50423382/f1798919-76f5-475f-a95e-09ddb59f82c9)

**Comment by wills106** [comment]
I have moved this to the discussions section as not an Integration issue.

Have you checked the Wiki where you need to set a setting different on the Waveshare compared to the SolaX Inverters.

> In "More Advanced Aettings...",
> 
> SolaX set Modbus Gateway Type to "Simple Modbus to TCP/IP mode"
> Sofar Solar set Modbus Gateway Type to "Auto query storage type"

If you are trying to connect both the Integration and Node to the Inverter than wont work in normal cases as you can't have two Master devices.

On the Soafar Inverters do you have a separate RS485 for the Meter and for coms for data logging?

**Comment by TwardyBobek** [comment]
Połączyłem się, chodziło o osobny port rs485 niż ten co urzywany jest do smartmiter. Sofar Solar ma razem 3 porty rs485 i  1 port can w jednej wtyczce a w Waveshare opcja Enable Multi-host musi być włączona. Podgląd działa ale  nadal nie mogę sterować falownikiem.
